contracts/core/management/VaultManager.sol

```diff
@@ -547,8 +547,10 @@ abstract contract VaultManager is
       // This check is necessary because if there is any rebase token or the protocol sets the balance to zero,
       // we need to be able to withdraw other tokens. The balance for a withdrawal should always be >0,
       // except when the user accepts to lose this token.
-      if (tokenBalance == 0 && _exemptionTokens[exemptionIndex] != _token)
-        revert ErrorLibrary.WithdrawalAmountIsSmall();
+      if (tokenBalance == 0) {
+        if (_exemptionTokens[exemptionIndex] != _token) exemptionIndex += 1;
```

Comment:
Here it should be like this, I think so, 
 if (tokenBalance == 0 && _exemptionTokens[exemptionIndex] != _token)
        revert ErrorLibrary.WithdrawalAmountIsSmall();
  else if(tokenBalance == 0 &&_exemptionTokens[exemptionIndex] == _token ){
        exemptionIndex += 1
}